lib/rules/no-descending-specificity/index.cjs

```diff
@@ -88,9 +88,11 @@ const rule = (primary, secondaryOptions) => {
 						continue;
 					}
 
-					parseSelector(resolvedSelector, result, ruleNode, (s) => {
-						checkSelector(resolvedSelector, s, ruleNode, comparisonContext);
-					});
+					const selectorRoot = parseSelector(resolvedSelector, result, ruleNode);
```

Comment:
major

I think that changing the signature of that function will break many plugins that use the callback argument.
It's under utils so it's public.
Let's wait for v17?

see https://github.com/search?q=%22stylelint%2Flib%2Futils%2FparseSelector%22&type=code